src/api/batch/batch-file.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: default
+grand_parent: API Reference
+parent: Batch
+title: Batch File
+permalink: /api/batch-file
```

Comment:
This documents the Farmlands card load batch format.  Should be like:

```yaml
parent: Batch Types
title: Farmlands Batch Type
permalink: /api/batch-types/farmlands
```

Reply:
@ncjones Could you please check out the updates I have made to the batch folder's structure and let me know what you think.